kickoff/static/treestatus.js

```diff
@@ -0,0 +1,49 @@
+function _isTreeForThunderbird(treeName) {
+    return treeName.indexOf('comm') > -1;
+}
+
+function getTreeStatusUrl(branchName) {
+    var treeName = branchName.split('/')[1];
+    if (_isTreeForThunderbird(treeName)) {
+        treeName += '-thunderbird';
+    }
+
+    return 'https://treestatus.mozilla.org/' + treeName + '?format=json';
```

Comment:
What about a config file for JS?
In case this URL changes (tests would have to be updated too)